src/Tribe/Attendees.php

```diff
@@ -820,6 +825,47 @@ public function maybe_generate_csv() {
 		}
 	}
 
+	/**
+	 * Admin requested a CSV export from the attendees table.
+	 * Generates the download and finishes the execution.
```

Comment:
I think this could be tweaked to better describe the function itself.

```suggestion
	 * Export a CSV containing all attendees.
	 *
	 * Called when an Admin requested a CSV export from the attendees table. Generates
	 * the download file and finishes the execution.
```